staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/vault/BUILD

```diff
@@ -0,0 +1,53 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
This package ought to be in `plugin`.  `k8s.io/apiserver/plugin/pkg/kmsprovider/vault`

Reply:
Done!